specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json

```diff
@@ -4953,73 +4950,39 @@
       },
       "x-ms-discriminator-value": "AzureEventHubs"
     },
-    "AzureTableParameter": {
+    "AzureLogAnalyticsParameter": {
       "required": [
-        "connectionString",
+        "clientId",
+        "clientSecret",
         "query",
-        "table"
+        "tenantId",
+        "workspaceId"
       ],
       "type": "object",
       "properties": {
-        "connectionString": {
-          "description": "Azure Table connection string",
-          "type": "string"
-        },
-        "table": {
-          "description": "Table name",
+        "tenantId": {
+          "description": "The tenant id of this Log Analytics",
```

Comment:
This description needs to be improved to more clearly indicate what the property is.

Reply:
Updated.